### DISCUSSION TITLE
Can not find Part1AdaptiveDialog folder

### DISCUSSION THREAD

**Discussion by sARASWAN-sINHA** [rvinothrajendran/BotTutorialSample#7] [General]
Please specify the path to Part1AdaptiveDialog folder.

**Comment by rvinothrajendran** [comment]
@sARASWAN-sINHA Part1AdaptiveDialog I think this one https://github.com/rvinothrajendran/BotTutorialSample/tree/master/AdaptiveDialog/Csharp/AdaptiveDialogInput ?
